compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/split/values/JvmRecordGen.java

```diff
@@ -866,4 +895,120 @@ private boolean checkIfValueIsJReferenceType(BType bType) {
                 return true;
         }
     }
+
+    void createUnboxedIntGetMethod(ClassWriter cw, Map<String, BField> fields, String className,
+                                   JvmCastGen jvmCastGen) {
+        createBasicTypeGetMethod(cw, fields, className, jvmCastGen, TypeKind.INT, "getUnboxedIntValue",
```

Comment:
As these are one-line methods, shall we call them directly instead of having a separate method?

Reply:
Fixed